### PR TITLE
Add Test Utilities and FreeScheduleCommandTest

### PR DESCRIPTION
**Pull request by hanqinilnix**
I have added Test Utilities for the events and schedule so that we can create test cases with more ease. At the same time, I have also completed one of the test class for a command that is not tested

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 1]
```diff
@@ -76,7 +98,7 @@
      * - the {@code actualModel} matches {@code expectedModel}
      */
     public static void assertCommandSuccess(Command command, Model actualModel, CommandResult expectedCommandResult,
-            Model expectedModel) {
+                                            Model expectedModel) {
```
Comment (tyanhan) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:101]: There seems to be some spacing here.

**Review by tyanhan — CHANGES_REQUESTED** [review]
Looks good! Just minor changes, great job.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -187,19 +189,26 @@ Format: `addEvent INDEX ed/EVENT_DESCRIPTION da/DATE [ti/TIME] [du/DURATION] [t/
 
 
 Examples:
-* `addSchedule 3 da/20-12-2022 ti/10:00 du/2H30M`
-* `addSchedule 2 da/22-11-2023 ti/12:00`
+* `addSchedule 3 da/2022-12-20 ti/10:00 du/2H30M`
```
Comment (markbiju) [inline docs/UserGuide.md:192]: Should the command should be addEvent here?

**Review by markbiju — COMMENTED** [review]
Good work, just a minor change

File: docs/UserGuide.md (hunk 2 above)
Comment (hanqinilnix) [reply docs/UserGuide.md:192]: you are right

**Review by tyanhan — APPROVED** [review]
LGTM!

**Pull request merged**